packages/core/useWebSocket/index.ts

```diff
@@ -223,7 +223,7 @@ export function useWebSocket<Data = any>(
     heartbeatResume = resume
   }
 
-  if (options.immediate) _init()
+  if (options.immediate !== false) _init()
```

Comment:
could just be `if (options.immediate) _init()` right? 😅

Reply:
No, cause the new default is that if it's `undefined` than it should also count as `true`
That's the whole point of this PR 😛

Reply:
My bad 😅

Reply:
Better to assign default value in L121 instead.